src/main/webapp/app/quiz/participate/quiz.component.ts

```diff
@@ -110,6 +111,7 @@ export class QuizComponent implements OnInit, OnDestroy {
         private jhiAlertService: JhiAlertService,
         private quizSubmissionService: QuizSubmissionService,
         private translateService: TranslateService,
+        public deviceService: DeviceDetectorService,
```

Comment:
I would try to avoid exposing services to the templates. I.m.o. the best practice would be to have some variable/getter/method exposing the information you need.

Reply:
That's a good point - I changed that :)